README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+Nabilah Sammudin
+7/6/2020
+
+## Brief Biography
+
+My name is **Nabilah** and I am from **Malaysia**. I graduated from **Purdue University** back in 2017 and I am currently working as a research technician in the same lab I did my research as an undergraduate. 
+
+I am interested in studying *Learning and Memory* at the *cell, molecular level* as well as potentially using *computational techniques* in my research. Although I have very little background in coding, I believe that it is an invaluable skill in doing my PhD and I am excited to learn more through this class!
+
+### Hobbies
+
+* Baking 
+* Cooking
+* Reading
+* Hiking
+
+I recently stumbled across this beautiful sunset picture and it is my current favorite image.
+
+![Sunset on the Rocks in Malaysia](https://asiasociety.org/files/uploads/303images/121108_POD_malaysia.jpg)
```

Comment:
I really like the picture! I've been having the same issue as Ariana and yourself with the image caption. I'll look at this some more as well, the guides Canvas linked seem to have used the exact same format as us.